.bcr/README.md

```diff
@@ -0,0 +1,163 @@
+1. Creates a new entry in the BCR with the release version
+2. Opens a pull request to the [bazel-central-registry](https://github.com/bazelbuild/bazel-central-registry) repository
+3. Allows BCR maintainers to review and merge the update
+
+This makes the new version of `fast_float` available to all Bazel users through the standard module system.
+
+## Files in This Directory
+
+### `metadata.template.json`
+Contains repository metadata including:
+- Project homepage
+- Maintainer information (name, email, GitHub username)
+- Repository references
+
+This information is used to populate the BCR entry's `metadata.json` file.
+
+### `source.template.json`
+Defines how to fetch the source code for each release:
+- URL pattern for release archives
+- Archive strip prefix configuration
+- Integrity hash (auto-generated during publishing)
+
+### `presubmit.yml`
+Specifies the CI tests that BCR will run before accepting a new version:
+- Platforms to test on (Linux, macOS, Windows)
+- Bazel versions to test with
+- Build targets to verify
+
+## How the Publishing Workflow Works
+
+The publishing process is automated through GitHub Actions:
+
+1. **Trigger**: When you create a new release on GitHub (or manually trigger the workflow)
+2. **Build**: The workflow uses [bazel-contrib/publish-to-bcr](https://github.com/bazel-contrib/publish-to-bcr) reusable workflow
+3. **Generate**: Creates BCR entry files from the templates in this directory
+4. **Submit**: Opens a pull request to the Bazel Central Registry
+5. **Review**: BCR maintainers review and merge the PR
+
+## Setting Up the Personal Access Token (PAT)
+
+The publishing workflow requires a GitHub Personal Access Token with appropriate permissions to:
+- Push to a fork of the Bazel Central Registry
+- Open pull requests against the upstream BCR
+
+### Creating a Personal Access Token
+
+1. **Navigate to GitHub Settings**
+   - Go to https://github.com/settings/tokens
+   - Or: Click your profile picture → Settings → Developer settings → Personal access tokens → Tokens (classic)
+
+2. **Generate New Token**
+   - Click "Generate new token (classic)"
+   - Give it a descriptive name, e.g., "BCR Publishing for fast_float"
+   - Set an appropriate expiration (recommended: 1 year with calendar reminder to renew)
+
+3. **Select Required Scopes**
+   - ✅ **`repo`** (Full control of private repositories) - Required for accessing repository details
+   - ✅ **`workflow`** (Update GitHub Action workflows) - Required for the publishing action
+
+4. **Generate and Copy**
+   - Click "Generate token" at the bottom
+   - **Important**: Copy the token immediately - you won't be able to see it again!
+
+5. **Add as Repository Secret**
+   - Go to the fast_float repository settings
+   - Navigate to: Settings → Secrets and variables → Actions
+   - Click "New repository secret"
+   - Name: `BCR_PUBLISH_TOKEN`
+   - Value: Paste the token you copied
+   - Click "Add secret"
```

Comment:
I'm not allowed to provide the token, this is a task for fast_float project maintainer, see this